app/renderer/components/LogsPanel/index.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import styled from 'react-emotion'
+import theme from '@/style/common'
+import moment from 'moment'
+
+const Box = styled.div` 
+  border: 1px solid ${theme.mainBorderColor}; 
+  background: ${theme.contrastColor}; 
+  box-shadow: 0 2px 1px rgba(0, 0, 0, 0.04), inset 0 -1px 2px rgba(0, 0, 0, 0.05); 
+  margin: 2rem; 
+ 
+  h1 {
+    padding: 1rem; 
+  }
+  
+  hr {
+    margin: 0 1rem 0.5rem 1rem;
+    display: block;
+    height: 1px;
+    border: 0;
+    border-top: 1px solid ${theme.mainBorderColor};
+    padding: 0; 
+  }
+  
+  ul {
+    list-style: none; 
+  }
+  
+  li {
+    display: flex;
+    align-items: center;
+  }
+`
+
+const Level = styled.div`
+  margin: 1rem;
+  width: 1rem;
+  height: 1rem;
+  border-radius: 50%;
+  background: ${props => {
+  if (props.value === 'info'){
+    return theme.info;
+  }
+  return theme.grayTextColor;
+}};
+`
+
+const Time = styled.div`
+  width: 6rem;
+  margin-left: 2rem;
+`
+
+const Message = styled.div`
+  flex: 1;
+  
+  > div {
+    margin: 0.5rem 0;
+  }
+`
+
+const Server =  styled.span`
+  color: ${theme.info};
+  font-style: italic;
+`
+
+const LogsPanel = ({records}) => <Box>
+  <h1>Recent log entries</h1>
+  <hr/>
+  <ul>
+    {records.map(({ id, level, message, timestamp, server }) =>
+      <li key={id}>
+        <Level value={level}/>
+        <Message>
+          <div>{message}</div>
+          <div>Posted By: <Server>{server}</Server></div>
+        </Message>
+        <Time>{ moment(timestamp).fromNow()}</Time>
```

Comment:
We will probably need it for charts and probably else where. Since this is native i don't think bringing it in costs much. If you still think we should remove it I will, not critical.